src/main/resources/GoogleApps/GoogleAppsConfigTemplate.xml

```diff
@@ -130,17 +159,64 @@
         <validationRegExp/>
         <classType>com.xpn.xwiki.objects.classes.PasswordClass</classType>
       </secret>
+      <skipLoginPage>
+        <customDisplay/>
+        <defaultValue/>
+        <disabled>0</disabled>
+        <displayFormType>checkbox</displayFormType>
+        <displayType/>
+        <name>skipLoginPage</name>
+        <number>8</number>
+        <prettyName>skipLoginPage</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </skipLoginPage>
+      <useCookies>
+        <customDisplay/>
+        <defaultValue/>
+        <disabled>0</disabled>
+        <displayFormType>checkbox</displayFormType>
+        <displayType/>
+        <name>useCookies</name>
+        <number>7</number>
+        <prettyName>useCookies</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </useCookies>
     </class>
-    <name>GoogleApps.GoogleAppsConfigTemplate</name>
-    <number>0</number>
-    <className>GoogleApps.GoogleAppsConfigClass</className>
-    <guid>bad4b5eb-14e0-4986-9e1c-9a7863d12faa</guid>
     <property>
-      <clientid>hash:SHA-512:cf83e1357eefb8bdf1542850d66d8007d620e4050b5715dc83f4a921d36ce9ce47d0d13c5d85f2b0ff8318d2877eec2f63b931bd47417a81a538327af927da3e</clientid>
+      <activate/>
+    </property>
+    <property>
+      <appname/>
+    </property>
+    <property>
+      <authWithCookies/>
+    </property>
+    <property>
+      <clientid/>
```

Comment:
I see that the value of the ``clientid`` property is lost. Was it needed only for development? Aren't users going to be affected?

Reply:
Yes. I'm fairly certain that is was for development only. The client id needs to be filled by the administrator anyway.